server/api/prospect_handler.py

```diff
@@ -43,39 +43,44 @@ def upload_prospects():
 
     # Add to db
     prospect_json = prospect_json.copy()
-    prospect_list = list(prospect_json.values())
+    prospect_list = list(prospect_json['prospects'].values())
+    owner_email = prospect_json['owner']
```

Comment:
This should be coming from the JWT identity (for security purposes, or anyone can pass any owner email)

Reply:
Resolved in PR #66